src/Pages/Channels.razor

```diff
@@ -153,7 +154,13 @@
                 </DataGridColumn>
                 <DataGridColumn TItem="Channel" Field="@nameof(Channel.CreationDatetime)" Caption="@nameof(Channel.CreationDatetime).Humanize(LetterCasing.Sentence)" Sortable="true"/>
                 <DataGridColumn TItem="Channel" Field="@nameof(Channel.UpdateDatetime)" Caption="@nameof(Channel.UpdateDatetime).Humanize(LetterCasing.Sentence)" Sortable="true"/>
-
+                <DataGridColumn TItem="Channel" Field="@nameof(Channel.ChanId)" Caption="Channel Id" Sortable="false">
+                    <DisplayTemplate>
+                        <a href="@(Constants.AMBOSS_ENDPOINT + "/search?term=" + context.ChanId)" target="_blank">
```

Comment:
Instead of `/search?term=`  I'd use `/edge/` and this concatenated with `context.ChanId` . 
This is an url example for a channel -> https://amboss.space/edge/813298855542849537

Reply:
Thanks! Didnt know that one